### PR TITLE
[Feature] add customer summary view

### PR DESCRIPTION
**Pull request by syedjaferk**

### CODE REVIEW

**Review by codered1234 — CHANGES_REQUESTED** [review]
There are no behaviours in the Address and Customer class. 
You are using this as a datastructure. 

Its a code smell !!!

**Comment by syedjaferk** [comment]
> There are no behaviours in the Address and Customer class. You are using this as a datastructure.
> 
> Its a code smell !!!

Will check

File: tool.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    def set_post_code(self, post_code):
+        self._post_code = post_code
+    
+    def get_country(self):
+        return self._country
+    
+    def set_country(self, country):
+        self._country = country
+    
+    def get_address_summary(self):
```
Comment (codered1234) [inline tool.py:25]: Good !

**Review by codered1234 — APPROVED** [review]
Good!
You have been saved from anemic models.

**Pull request merged**